cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -5,7 +5,20 @@ service:
       receivers: [otlp, jaeger, zipkin]
       processors: [batch]
       exporters: [jaeger_storage_exporter]
-
+  telemetry:
+    resource:
+      service.name: jaeger
+    metrics:
+      level: detailed
+    # TODO Initialize telemetery tracer once OTEL released new feature.
```

Comment:
When leaving TODO comments it's best to link to an issue that provides more background & details. And if you had an issue you could just have the URL here instead of the commented out YAML (which can be mentioned in the issue).